.dockerignore

```diff
@@ -0,0 +1 @@
+.token
```

Comment:
I just realized important additions for this would also be:
- `bin` to skip built backends
- `node_modules` for consistency

For CI it shouldn't matter, but on my dev machine it reduced the build context from 291.4MB to 14.08MB.